### DISCUSSION TITLE
When I paste a command into the command line, the characters appear one by one instead of all at once.

### DISCUSSION THREAD

**Discussion by swpper** [ohmyzsh/ohmyzsh#12527] [Help] [answered]
Under the same configuration, it works normally in the MacOS command line, but it doesn't work properly in Linux. When I comment out the "source $ZSH/oh-my-zsh.sh" line in my zshrc file, it works properly.
The theme I used is 'powerlevel10k/powerlevel10k', and plugin are
plugins=(
  git
  zsh-syntax-highlighting
  zsh-autosuggestions
)
no more custom configuration.

**Comment by mcornella** [comment, marked as answer]
This is probably happening because of `zsh-syntax-highlighting` plugin. You can try updating the plugin or using the https://github.com/zdharma-continuum/fast-syntax-highlighting plugin.

**Comment by swpper** [comment]
Hello, I have received your message in my email and will response quickly, thank you.
